config/initializers/git_sha.rb

```diff
@@ -3,8 +3,8 @@
 revisions_logfile = Rails.root.join("..", "..", "revisions.log")
 
 GIT_SHA =
-  if File.exist?(revisions_logfile)
-    `tail -1 #{revisions_logfile}`.chomp.split(" ")[3].gsub(/\)$/, '')
+  if File.exists?("REVISION")
```

Comment:
`REVISION` is a local file and should always have the right information. Unlike `../../revisions.log` that gets updated at the end of the deployment process.